pkg/daemon/ceph/client/upgrade.go

```diff
@@ -104,6 +157,197 @@ func EnableNautilusOSD(context *clusterd.Context) error {
 		return fmt.Errorf("failed to disallow pre-nautilus osds and enable all new nautilus-only functionality: %+v", err)
 	}
 	logger.Infof("successfully disallowed pre-nautilus osds and enabled all new nautilus-only functionality")
+	return nil
+}
+
+// OkToStop determines if it's ok to stop an upgrade
+func OkToStop(context *clusterd.Context, namespace, deployment, clusterName string, cephVersion cephver.CephVersion) error {
```

Comment:
I don' think `client.OkToStop` is very clear. Someone might wonder, "okay to stop what?". I think this should be something more like `OkToStopDaemon`.

Reply:
bump